compiler-utils/src/testFixtures/java/com/squareup/anvil/compiler/internal/testing/AnvilCompilation.kt

```diff
@@ -36,11 +37,14 @@ public class AnvilCompilation internal constructor(
     generateDaggerFactoriesOnly: Boolean = false,
     disableComponentMerging: Boolean = false,
     enableExperimentalAnvilApis: Boolean = true,
+    codeGenerators: List<CodeGenerator> = emptyList(),
   ) = apply {
     checkNotCompiled()
     anvilConfigured = true
     kotlinCompilation.apply {
-      compilerPlugins = listOf(AnvilComponentRegistrar())
+      compilerPlugins = listOf(
+        AnvilComponentRegistrar().also { it.addCodeGeneratorsManually(codeGenerators) }
```

Comment:
Super nit, but it feels implied that the code generators are included manually, so I would drop the "manually" nomenclature. Just keep it as `addCodeGenerators(codeGens)`.

Reply:
Done